Description/DescriptionEnricherInterface.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2015 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Component\Resource\Description;
+
+use Symfony\Cmf\Component\Resource\Repository\Resource\CmfResource;
+
+interface DescriptionEnricherInterface
```

Comment:
Or use Enhancer as with the RoutingBundle - I think the semantic is the same /cc @dbu